### ISSUE TITLE
Installer broken on linux

### ISSUE THREAD

**Issue by UltraBlackLinux** [soler91/noita-together#77]
Hey there,
I'm having some trouble with the linux installer: I can't select the .exe file. 
Can you fix this?
Thanks!

**Comment by PackmanDude** [comment]
Nice pfp. Have you tried running AppImage?

**Comment by PackmanDude** [comment]
Nice pfp. Have you tried selecting the game's root folder instead of an executable?

**Comment by UltraBlackLinux** [comment]
(Thanks lol) 
You have to select the exe afaik

**Comment by PackmanDude** [comment]
> You have to select the exe afaik

No, you don't have to. Maybe you misunderstood, but it should search for Noita's location at the default GOG/Steam location, and if it can't, you can provide it manually. It also works well on Linux for me; re-read the instructions in the README and on the release page. It looks complex, but it is not that hard.
